GroupBundle/DataFixtures/MongoDB/LoadGroupDemoData.php

```diff
@@ -18,9 +20,27 @@ class LoadGroupDemoData extends AbstractLoadGroupData implements OrchestraProduc
     public function load(ObjectManager $manager)
     {
         $group2 = $this->generateGroup('Demo group', 'Demo group', 'Groupe de démo', 'site2', 'group2');
-        $group2->addRole(AdministrationPanelStrategy::ROLE_ACCESS_REDIRECTION);
         $manager->persist($group2);
 
+        $adminGroup = $this->generateGroup('Admin group', 'Admin group', 'Groupe admin', null, 'group-admin');
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_REMOVED_TRASHCAN);
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_THEME);
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_CREATE_THEME);
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_UPDATE_THEME);
+        $adminGroup->addRole(AdministrationPanelStrategy::ROLE_ACCESS_DELETE_THEME);
+        $adminGroup->addRole(LogPanelStrategy::ROLE_ACCESS_LOG);
+        $adminGroup->addRole(TreeNodesPanelStrategy::ROLE_ACCESS_ERROR_NODE);
+        $adminGroup->addRole(TreeNodesPanelStrategy::ROLE_ACCESS_CREATE_ERROR_NODE);
+        $adminGroup->addRole(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_ERROR_NODE);
+
+        if ($this->hasReference("group-folders")) {
```

Comment:
no you should make like in AbstractLoadGroupData of workflow function data, it has to be mediaAdminBundle than add role on adminGroup